gateway/src/main/java/com/bakdata/quick/gateway/directives/topic/rule/validation/ValidationUtility.java

```diff
@@ -0,0 +1,65 @@
+/*
+ *    Copyright 2022 bakdata GmbH
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.bakdata.quick.gateway.directives.topic.rule.validation;
+
+import com.bakdata.quick.common.graphql.GraphQLUtils;
+import com.bakdata.quick.gateway.directives.topic.TopicDirectiveContext;
+import graphql.language.InputValueDefinition;
+import graphql.language.ObjectTypeDefinition;
+import java.util.Objects;
+import java.util.Optional;
+
+public class ValidationUtility {
```

Comment:
We always call them Utils so far:
```suggestion
public final class ValidationUtils {
```